app/imageUpload/imageController.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var fs = require('fs');
+var imgur = require('imgur');
+var multer = require('multer');
+
+var upload = multer({
+  dest: 'uploads/',
+  limits: {
+    fileSize: 10000000 //20MB
+  },
+  fileFilter: function (req, file, cb) {
+    cb(null, file.mimetype.slice(0,6) === 'image/');
+  }
+});
+
+var uploadMulter = upload.single('image')
+
+module.exports.img = function (req, res, next) {
```

Comment:
Can you come up with a better function name than "img"? Even `module.exports = function` would be preferred. Just change the routes.js file to match.